exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,38 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {    
+  // return number < 0; //(This works)
+
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+  // if (number < 0){
+  //   return typeof number == "negative";
+  // }else{
+  //   return typeof number == "positive";
+  // }  //(I don't know why this does not work)
```

Comment:
Because `typeof` returns the javascript data type of the thing, and "positive" and "negative" are not javascript data types :) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#javascript_types

Reply:
Also, hm, on second look I've noticed this is already inside an if statement -- what were you trying to achieve here?